config/settings.yml

```diff
@@ -21,3 +21,11 @@ plugins:
     default: false
   discourse_jira_sync_issue_comments:
     default: false
+  discourse_jira_allowed_groups:
+    default: "1" # @admins
+    mandatory_values: "1"
+    type: group_list
+    client: true
```

Comment:
No need to send group settings like this to the client, it's pointless to compare on the client since we don't always serialize all the user's groups to the client. You should add checks to the current user serializer, which you did already in this PR, so this can just be removed :)

Reply:
okay solid! I wasn't positive on `client` either way, but that makes sense.